### PR TITLE
Changes in NN stream aggregator.

### PR DESCRIPTION
**Pull request by cfortuna**

### CODE REVIEW

File: src/glib/mine/anomaly.cpp [hunk 1]
```diff
@@ -156,38 +156,43 @@ PJsonVal TNearestNeighbor::Explain(const TIntFltKdV& Vec) const {
...
-        DiffVal->AddToArr(FtrVal);
-        // move to the next feature
-        if (NearVec[NearEltN].Key > Vec[EltN].Key) {
-            EltN++;
-        } else if (NearVec[NearEltN].Key < Vec[EltN].Key) {
-            NearEltN++;
-        } else {
-            NearEltN++; EltN++;
+        //avoid unnecessary fields in the explanation
+        if (Diff < 1e-8) {
```
Comment (blazf) [inline src/glib/mine/anomaly.cpp]: Should be `Diff > 1e-8`, not `<`

Comment (cfortuna) [inline src/glib/mine/anomaly.cpp]: Done.

File: src/nodejs/qm/qm_nodejs_streamaggr.h [hunk 2]
```diff
@@ -1040,6 +1040,82 @@
...
+*             { name: 'Temperature', type: 'float' },
+*             { name: 'Precipitation', type: 'float' },
+*             { name: 'Time', type: 'datetime' }
+*         ]
+*     }]
+* });
+* // create the store
+* let store = base.store('Cars');
+* //define a feature space aggregator on the Cars store which needs at least 2 records to be initialized. Use three of the 
+* four fields of the store to create feature vectors with normalized values.
```
Comment (cfortuna) [inline src/nodejs/qm/qm_nodejs_streamaggr.h]: Done.

**Comment by rupnikj** [comment]
One additional thing - in the example you use ES6 features `let base...` - this won't be ok for node.js 0.12 (we test examples, although this might be currently disabled)

**Pull request merged**